circletree/CircleSegment.jsx

```diff
@@ -17,7 +17,8 @@ var defaultProps = {
   total: 1,
   // leaf settings
   leafRadius: 7,
-  leafSpacing: 2
+  leafSpacing: 2,
+  filter: {}
```

Comment:
if we can add a `// optional key:value object for filtering which leaves to _not_ render`, that'll probably improve future understanding